DOCS.md

```diff
@@ -166,13 +166,37 @@ pipeline:
     # ...
 ```
 
+### `wait_rollouts`
+
+_**type**_ `[]string`
+
+_**default**_ `[]`
+
+_**description**_ wait for the listed rollouts using `kubectl rollout status ...`
+
+_**example**_
+
+```yaml
+# .drone.yml
+---
+pipeline:
+  # ...
+  deploy:
+    image: nytimes/drone-gke
+    wait_rollouts:
+    - deployment/nginx
+    - deployment/memcache
+    - statefulset/hazelcast
+    # ...
+```
+
 ### `wait_deployments`
```

Comment:
Can you deprecate this field and tell people to use `wait_rollouts`?

Reply:
We don't need `wait_rollouts` but I'm not sure how to deprecate in a way that won't cause a lot of disruption for teams.